tools/db_bench_tool.cc

```diff
@@ -3835,6 +3868,32 @@ void VerifyDBFromDB(std::string& truth_db_name) {
       if (s.ok()) {
         db->db = ptr;
       }
+    } else if (FLAGS_use_secondary_db) {
```

Comment:
how about
```
} else if (FLAGS_use_secondary_db && FLAGS_secondary_update_interval > 0) {
```

Reply:
Even if `FLAGS_secondary_update_interval <= 0`, I think we can still open the DB as secondary instance. We just do not start the update thread. Will update